beta-src/src/models/BoardClass.ts

```diff
@@ -0,0 +1,83 @@
+import { IBoard } from "./Interfaces";
+import TerritoryClass from "./TerritoryClass";
+import UnitClass from "./UnitClass";
+
+// this is will be used for accessing all necessary data and methods
+
+export default class BoardClass {
+  // all territories.
+  territories: TerritoryClass[];
+
+  // includes entire units currently placed in the board.
+  units: UnitClass[];
+
+  // This will be used for order generation.
+  myUnits: UnitClass[];
+
+  constructor({ territories, units, terrStatus }: IBoard) {
```

Comment:
alpha order. also is terrStatus territoryStatus? if so, why isn't this part of the each territory in `territories`?

Reply:
terrStatus is coming from different api calls. Class will receive both territories(default data) and terrStatus(game specific) and combine. I've made a class to receive un-touched and process the data. Is it better to process the data after API call and feed class with it?